gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/AuthConfig.groovy

```diff
@@ -55,6 +56,14 @@ class AuthConfig {
   @Autowired
   FiatPermissionEvaluator permissionEvaluator
 
+  @Autowired
+  FiatSessionFilter fiatSessionFilter
+
+  @Bean
+  FiatSessionFilter fiatSessionFilter() {
```

Comment:
I think this bean creation will add the filter to the original filter chain, thus adding it twice.